packages/scalecube-microservice/src/Dispatcher/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Message } from '../api/Message';
+
+export const createDispatcher = ({ router, serviceRegistry, preRequest$, postResponse$ }) => {
+  const routerInstance = new router(serviceRegistry);
```

Comment:
Where do you use routerInstance in this scope?

Reply:
TODO the "systemCall" is using the routerInstance